react/features/base/devices/middleware.js

```diff
@@ -47,6 +31,25 @@ const JITSI_TRACK_ERROR_TO_MESSAGE_KEY_MAP = {
  */
 // eslint-disable-next-line no-unused-vars
 MiddlewareRegistry.register(store => next => action => {
+    // If defined in the global scope, it causes ReferenceError for JitsiTrackErrors. It seems that circular
```

Comment:
Please don't do this. Fix the circular dependency error instead of introduccing a hack.

Reply:
What do you mean "Fix the circular dependency error"? There's no "circular dependency error"! We should either remove the circular dependancy or either have this solution. 

Just FYI because of how we do our imports (importing the whole feature and not the file) we have a lot of circular dependancies :) The problem occurs when we are defining stuff on the global scope of the file only because in this case they are `undefined`.